apps/web/src/app/[locale]/(dashboard)/[wsId]/tasks/boards/components/GanttTimeline.tsx

```
@@ -0,0 +1,373 @@
+            ))}
+          </div>
+        </div>
+      </div>
+
+      {/* Gantt Chart - Fixed Height Container */}
+      <div
+        className="relative rounded-lg border bg-background"
+        style={{ height: '320px' }} // Fixed height instead of expanding
+      >
+        <style
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)